addons/mail/static/src/models/messaging_initializer.js

```diff
@@ -12,6 +12,9 @@ registerModel({
         async performInitRpc() {
             return await this.messaging.rpc({
                 route: '/mail/init_messaging',
+                params: {
+                    context: this.env.services.user.context,
```

Comment:
Hello @seb-odoo ,

To be sure, can we assume user service is in the environment or should we use the session by importing `@web/session`? 

Thanks in advance! 🙂 

Have a nice day! 🙂

Reply:
Hello, this is more a question for framework js team, I don't remember exactly as it changed a lot between versions.

But if it's there when you test it, I guess it will always be there?

Reply:
I guess yes, it was just to be sure to avoid having any crash in stable. 🙂 

But okay, I will ask to Framework JS to be sure before merging this fix, thanks. 

Have a nice afternoon! 🙂

Reply:
If you want to make sure, also double chat livechat and discuss channel public page (invite link).